### PR TITLE
Feature/website generating components properties

### PR DESCRIPTION
**Pull request by anna-lach**
#252 

Readme file aligned with the changes

### CODE REVIEW

File: docs/website/src/site/categories/components/component-table.njk [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  <div class="ds-table-wrapper">
+    <table class="ds-table"><thead>
+      <tr>
+        <th>Name</th>
+        <th>Type</th>
+        <th>Default</th>
+        <th class="ds-table__description">Description</th>
+      </tr></thead>
+      <tbody>
+      {% for member in declaration.members -%}{% if member.privacy %}{% else %}{% if member.kind == "field" %}<tr>
```
Comment (Diaan) [inline docs/website/src/site/categories/components/component-table.njk:18]: this looks like an empty if statement, can't you switch the assertion around?

File: docs/website/src/site/categories/components/component-table.njk [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  <div class="ds-table-wrapper">
+    <table class="ds-table"><thead>
+      <tr>
+        <th>Name</th>
+        <th>Parameters</th>
+        <th>Return</th>
+        <th class="ds-table__description">Description</th>
+      </tr></thead>
+      <tbody>
+      {% for member in declaration.members -%}{% if member.privacy %}{% else %}{% if member.kind == "method" %}<tr>
```
Comment (Diaan) [inline docs/website/src/site/categories/components/component-table.njk:43]: this looks like an empty if statement, can't you switch the assertion around?

**Review by Diaan — APPROVED** [review]

File: docs/website/src/site/categories/components/component-table.njk (hunk 1 above)
Comment (anna-lach) [reply docs/website/src/site/categories/components/component-table.njk:18]: The negation was not working and I couldn't find a better solution in this case.

File: docs/website/src/site/categories/components/component-table.njk (hunk 2 above)
Comment (anna-lach) [reply docs/website/src/site/categories/components/component-table.njk:43]: The negation was not working and I couldn't find a better solution in this case.

File: docs/website/custom-elements-manifest.config.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  /** Globs to analyze */
+  globs: [
+    './../../packages/core/src/**/*.ts',
+    './../../packages/editor/src/**/*.ts',
+    './../../packages/grid/src/**/*.ts'
+  ],
+  /** Directory to output CEM to */
+  outdir: 'src/site/_data/custom-elements',
+  /** Run in dev mode, provides extra logging */
+  dev: true,
```
[the remark this answers is not in the document]
Comment (anna-lach) [reply docs/website/custom-elements-manifest.config.js]: Actually we don't need it now, maybe in the future if we will need more information we can always set this option to `true`. I've just set it to `false`.

File: docs/website/src/site/categories/components/component-table.njk (hunk 2 above)
Comment (anna-lach) [reply docs/website/src/site/categories/components/component-table.njk]: Changed :)

File: docs/website/src/site/categories/components/component-table.njk (hunk 1 above)
Comment (anna-lach) [reply docs/website/src/site/categories/components/component-table.njk]: Changed :)

**Pull request merged**